pkg/handlers/generic/mutation/imageregistries/credentials/inject_test.go

```diff
@@ -354,7 +354,10 @@ var _ = Describe("Generate Image registry patches", func() {
 						},
 					),
 				},
-				RequestItem: request.NewKubeadmConfigTemplateRequest("", "test-kubeadmconfigtemplate"),
+				RequestItem: request.NewKubeadmConfigTemplateRequest(
+					"",
+					"test-kubeadmconfigtemplate",
+				),
```

Comment:
(nit: Was this an automatic formatting change? I find it distracting, because this file is unrelated to the PR)

Reply:
Yes it was. Need to figure out why these changes aren't getting picked up on original pr.